params/superchain.go

```diff
@@ -65,19 +65,20 @@ func LoadOPStackChainConfig(chainID uint64) (*ChainConfig, error) {
 		ArrowGlacierBlock:             common.Big0,
 		GrayGlacierBlock:              common.Big0,
 		MergeNetsplitBlock:            common.Big0,
-		ShanghaiTime:                  nil,
+		ShanghaiTime:                  chConfig.CanyonTime, // Shanghai activates with Canyon
 		CancunTime:                    nil,
 		PragueTime:                    nil,
 		BedrockBlock:                  common.Big0,
 		RegolithTime:                  &genesisActivation,
+		CanyonTime:                    chConfig.CanyonTime,
```

Comment:
Can we pull this from the superchain config? (with override per chain if necessary), to avoid duplicate configuration?

Reply:
What do you mean by this? `chConfig` is loaded from the superchain registry on line 40.